frontend/src/sdk/abis.ts

```diff
@@ -193,7 +193,35 @@ export const ERC20_ABI = [
       },
     ],
     payable: false,
-    stateMutability: 'pure',
+    stateMutability: 'view',
```

Comment:
q: is this duplicated?

Reply:
pure or view doesn't affect much here, changed it to view for consistency